.github/workflows/ubuntu.yml

```diff
@@ -39,6 +39,14 @@ jobs:
             elixir-version: '1.14.3'
             os: 'ubuntu-latest'
             version-type: 'strict'
+          - otp-version: '27.0'
```

Comment:
```suggestion
          - otp-version: '27'
```

Reply:
This way we make it choose the latest `27.x.y.z...`, not the latest `27.0.y.z...` 😄

Reply:
Applied in https://github.com/erlef/setup-beam/pull/271/commits/d2ad6440a8f7071bde167f803e52e9a9a9df10e8